packages/shortcuts-extension/schema/plugin.json

```diff
@@ -256,55 +256,55 @@
       },
       "type": "object"
     },
-    "notebook:change-to-cell-heading-1": {
+    "notebook:change-cell-to-heading-1": {
```

Comment:
Can you leave the names of the shortcuts the same, and just change the "command" lines, so that if someone overrode the setting, it still is overridden? We'll change the name of the shortcuts in the 0.33 release, since it's a backwards-incompatible change.

Reply:
That way we can release this fix in a patch release to 0.32, rather than having to wait until 0.33.

Reply:
On second thought, we're already merging changes for 0.33, so I think this should just go in as-is. We can back a backport that is backwards compatible for the 0.32 series.

Reply:
Thank you for the suggestions and clarifications. Sorry that I did not fully understand your comments in the issue.

Working on it.

Reply:
I think we're good. I can make a similar change to the 0.32 branch once we have that set up so it can come out before 0.33. Sorry for the confusion.

Reply:
No worries at all. I was worried about causing you troubles.

Learned a lot. I found it is very necessary for me to review the basics of formal software project management, in order to better contribute on open source projects.

Thank you!